packages/ui/src/components/editor/panels/Viewport/overlay/LoadingOverlay.tsx

```diff
@@ -0,0 +1,33 @@
+import { useHookstate } from '@etherealengine/hyperflux'
+import React, { useEffect } from 'react'
+
+interface LoadingOverlayProps {
+  isLoading: boolean
+}
+
+const LoadingOverlay: React.FC<LoadingOverlayProps> = ({ isLoading }) => {
```

Comment:
Why you need a new control and why not use `LoadingView` control? Is there a particular usecase that can not be handled by LoadingView?

Reply:
refactored to use `LoadingView`, thanks